frontend/apps/app/components/SessionsNewPage/SessionsNewPage.tsx

```diff
@@ -26,9 +22,7 @@ export const SessionsNewPage: FC<Props> = ({ projectId }) => {
       headers: {
         'Content-Type': 'application/json',
       },
-      body: JSON.stringify({
-        projectId,
-      }),
+      body: JSON.stringify({}),
```

Comment:
With this PR, it is no longer possible to create a design session with a projectId set. Create a UI for SessionsNewPage in the next PR to allow selection of projects.

Reply:
👍